pkg/limatmpl/embed.go

```diff
@@ -552,11 +555,70 @@ func (tmpl *Template) combineNetworks() {
 	}
 }
 
+// yamlfmt will fail with a buffer overflow while trying to retain line breaks if the line
+// is longer than 64K. We will encode all text files that have a line that comes close.
+const maxLineLength = 65000
```

Comment:
Can we cover this threshold in the unit test?

Reply:
I thought it was already covered by the "long line" test. I've modified it to check against a smaller threshold to make it maybe simpler.